authors/apps/profiles/views.py

```diff
@@ -188,6 +215,8 @@ def get(self, request, username):
                 "first_name": profile.first_name,
                 "last_name": profile.last_name,
                 "bio": profile.bio,
+                "following": True,
```

Comment:
Does this mean I follow all users that follow me?  can you implement a custom function to check the following status here?

Reply:
@oluwagbenga-joloko okay let me work on that

Reply:
@oluwagbenga-joloko I  have added  a function to check for the following status